lib/src/Uncertainty/Distribution/TruncatedDistribution.cxx

```diff
@@ -338,54 +325,72 @@ Scalar TruncatedDistribution::computeScalarQuantile(const Scalar prob,
 Point TruncatedDistribution::getParameter() const
 {
   Point point(distribution_.getParameter());
-  point.add(finiteLowerBound_ ? 1.0 : 0.0);
-  point.add(lowerBound_);
-  point.add(finiteUpperBound_ ? 1.0 : 0.0);
-  point.add(upperBound_);
+  for (UnsignedInteger k = 0; k < getDimension(); ++ k)
+    point.add(bounds_.getFiniteLowerBound()[k] ? 1.0 : 0.0);
+  point.add(bounds_.getLowerBound());
+  for (UnsignedInteger k = 0; k < getDimension(); ++ k)
+    point.add(bounds_.getFiniteUpperBound()[k] ? 1.0 : 0.0);
+  point.add(bounds_.getUpperBound());
```

Comment:
There was a bug in the 1D truncated distribution. Either the infinite bounds are part of the parameters (ie flags+values) and compute[PDF|CDF]Gradient is wrong, or the values of the infinite bounds are not part of the parameters, and they should not appear in the list of parameters.

Reply:
I propose to exclude the finite flags and the infinite values so that the number of parameters is consistent with pdfgradient.

Reply:
ok